### DISCUSSION TITLE
Save files only in /output

### DISCUSSION THREAD

**Discussion by Magisdn** [jlesage/docker-jdownloader-2#180] [Q&A]
Hi!
How to specify in the settings that files should be saved to /output? In the General settings that is exactly how it is set, but when I add a link through the browser plugin the file is downloaded to /output/package_name.

**Comment by jlesage** [comment]
What is the value you set for the output folder in General Settings ?

**Reply by jlesage** [reply]
Ok, so it's the `package_name` part that you don't want ?  I'm not sure if this can be changed in JD...  Maybe you can also ask on the JD's forum for a better answer.
